tests/test_session.py

```diff
@@ -169,7 +169,9 @@ def test_create_session_from_server_info_file_with_wrong_password(
     server_info_file.write_text(f"{ip}:{port}\n1234")
     with pytest.raises(RuntimeError):
         session = BaseSession.create_from_server_info_file(
-            server_info_filepath=str(server_info_file), cleanup_on_exit=False
+            server_info_filepath=str(server_info_file),
+            cleanup_on_exit=False,
+            start_timeout=2,
         )
```

Comment:
Reduced the timeout from this test, as it was always "succeeding" when timing out after 100 seconds every time. This is related to the general exception catcher that was hiding an expected wrong password exception, see changes to `health_check.py` file in this commit.

As no actual Fluent session or client is being launched, this test should ideally fail instantly due to the wrong password.